backend/setup.py

```diff
@@ -0,0 +1,68 @@
+"""Installer for the collective.addon package."""
```

Comment:
This file shows its age from copy-pasta. We should port these fixes back into the source, whatever that is.

Also can we use cookiecutter to generate this?